docs/modules/reference/pages/service-assurance/monitors/DeviceConfigMonitor.adoc

```diff
@@ -0,0 +1,57 @@
+
+= DeviceConfigMonitor
+
+This monitor tries to retrieve configuration from the network device and shows the status of that retrieval (success, failed, and so on).
+
+== Monitor Facts
+
+[cols="1,7"]
+|===
+| Class Name
+| `org.opennms.features.deviceconfig.monitors.DeviceConfigMonitor`
+|===
+
+== Configuration and Use
+
+.Monitor specific parameters for the DeviceConfigMonitor
+[cols="1,3,1"]
+|===
+
+| Parameter
+| Description
+| Default value
+
+3+| *Required*
+
+| script-file
+| Script file to use to retrieve configuration.
+| `-`
+
+3+| *Optional*
+
+| config-type
+| If device has multiple configurations, specify the config-type with each service.
+| default
+
+| schedule
+| Schedule to fetch configuration, in the form of cron expression.
+| 0 0 0 * * ?
+
+|===
+
+
+== Examples
+
+Example for how to configure the DeviceConfigMonitor in the `poller-configuration.xml`.
+
+[source, xml]
+----
+<parameter key="schedule" value="${requisition:dcb:schedule|0 0 0 * * ?}"/>
+<parameter key="script-file" value="${requisition:script-file|juniper-vsrx-default}.dcb"/>
+----
+
+== Schedule and interval
+
+The Device Config Monitor service uses an the interval of 300secs and a cron expression for the schedule.
```

Comment:
The Device Config Monitor service uses an interval of 300secs and a cron expression for the schedule.